app/assets/javascripts/pageflow/editor/views/change_theme_view.js

```diff
@@ -0,0 +1,35 @@
+pageflow.ChangeThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'templates/change_theme',
+
+  ui: {
+    changeThemeButton: '.change_theme',
+    labelText: 'label .name'
+  },
+
+  events: {
+    'click .change_theme': function() {
+      pageflow.ChangeThemeDialogView.open({
+        model: this.model,
+        themes: this.options.themes
+      });
+    }
+  },
+
+  onRender: function() {
+    this.ui.labelText.text(this.labelText());
+  },
+
+  labelText: function() {
+    return this.options.label || this.labelPrefix() + ': ' + this.localizedThemeName();
```

Comment:
Duplication from a code/data perspective only. It makes a big difference for some users whether they can change something from a dropdown without going another layer of nesting deeper (feature 1), and it makes a big difference for some users whether they can preview graphical features of their themes in one place (feature 2).

Be that as it may, I shall go with your recommendation because it is simpler